keps/sig-node/4381-dra-structured-parameters/README.md

```diff
@@ -1212,6 +1266,33 @@ const ResourceSliceMaxAttributesAndCapacitiesPerDevice = 32
 // domain prefix are assumed to be part of the driver's domain. Attributes
 // or capacities defined by 3rd parties must include the domain prefix.
 //
+// The Kubernetes project reserves the "k8s.io/" domain prefix for standardizing
+// attributes and capacities. DRA drivers **SHOULD** use these standardized
+// names if they define a characteristic of a device that matches the intent of
+// a standard attribute (or capacity) name. This ensures consistency and
+// interoperability across different drivers when conveying the same idea.
+//
+// Currently, the two standard attributes are:
+//
+//  1. `k8s.io/cpuSocketNumber`: An integer value referring to the logical
+//     identifier assigned by the operating system for the physical CPU socket
+//     that a device is associated with. For a PCIe device, DRA drivers can
+//     determine this value by first reading its associated NUMA node from
+//     `/sys/bus/pci/devices/<PCI_ADDRESS>/numa_node`. Then, using that NUMA
```

Comment:
cc @gauravkghildiyal 
AFAIK, ₩/sys/bus/pci/devices/<PCI_ADDRESS>/numa_node₩ indicates the NUMA node index, not the physical socket index.
The NUMA node index will differ from the socket index if SNC or NPS is configured.
I'm not sure which sysfs interface provides accurate socket information. 
PPL could rely on external libraries like hwloc or libpci (hwloc internally uses libpci).